conda/multisheller/ergocub-software_activate.msh

```diff
@@ -0,0 +1,27 @@
+if_(is_set("COMSPEC")).then_([
+	sys.list_append("YARP_DATA_DIRS", path.join(env("CONDA_PREFIX"), "Library\\share\\ergoCub")),
+	sys.list_append("ROS_PACKAGE_PATH", path.join(env("CONDA_PREFIX"), "Library\\share")),
+	sys.list_append("AMENT_PREFIX_PATH", path.join(env("CONDA_PREFIX"), "Library")),
```

Comment:
Just FYI, did does not do anything until you install the package.xml (for ROS1) or an appropriate file in ament's resource_index. Anyhow, they do not hurt for sure.